lib/aha-api/connection.rb

```diff
@@ -16,9 +16,12 @@ def connection(options={})
         options.merge!(:proxy => proxy)
       end
 
+      accept = options.delete(:accept)
+
       # TODO: Don't build on every request
       Faraday.new(options) do |builder|
         builder.request :json
+        builder.response :json if accept == 'application/json'
```

Comment:
Without this, Faraday won't parse the response into a Hashie object as it did before.